boards/common/particle-mesh/reset.c

```diff
@@ -0,0 +1,54 @@
+/*
+ * Copyright (C)  2020 Christian Amsüss
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser
+ * General Public License v2.1. See the file LICENSE in the top level
+ * directory for more details.
+ */
+
+/**
+ * @{
+ *
+ * @brief       Management of bootloader resets
+ *
+ * This implements a reset procedure read from the code of the particle device
+ * OS at https://github.com/particle-iot/device-os/ (version
+ * c21d5b8f39c8a013e19f904800808673f369c6e3), primarily from
+ * hal/inc/syshealth_hal.h and its linker script.
+ *
+ * @file
+ * @author      Christian Amsüss <chrysn@fsfe.org>
+ *
+ * @}
+ */
+
+#ifdef MODULE_USB_BOARD_RESET
```

Comment:
No need for that, if `usb_board_reset_in_bootloader()` is not used the linker will just throw it out.

Reply:
Thanks, fixup'd